docs/api/rpc-params.md

```diff
@@ -0,0 +1,158 @@
+---
+id: rpc-params
+title: Named Parameters
+sidebar_label: Named Parameters
+---
+
+We are heading away from the positional-based APIs for easier maintainability.
+
+<blockquote class="warning">
+<strong>heads up</strong><br><br>
+
+This page refers to a **technical preview**.
+
+These features are not currently available on TestNet.  They will be available in an upcoming release.
+
+</blockquote>
+
+This is an improvement of our RPC API that allows querying account info, access key(s), and state by a given block identifier (either block height or block hash).
+
+## Introduction
+
+The following identifiers are used throughout the code samples below. 
+
+- `account_id`: (string) NEAR [account ID format](/docs/concepts/account)
+- `prefix`: (base64 string) optional string used to segment our key-value store
+- `public_key` (string) the public part of your public/private key pair
+- `method_name`: (string) the name of the RPC method you're calling
+
+## Common Parameters
+
+- `block_id`: (integer) block height or block hash
+- `finality`: (string) filter for status of block finalization [ `optmistic` | `near-final` | `final` ] 
+
+
+*At time of writing, `finality` is only available on StageNet.*
+
+To run the examples below, you will likely need a more recent block height or block hash.  You can find that here: https://staging-rpc.nearprotocol.com/status
+
+### Using block height
+
+```bash
+http post https://staging-rpc.nearprotocol.com jsonrpc=2.0 id=dontcare \
+    method=query \
+    'params:={
+                "request_type": "view_access_key_list", 
+                "account_id": "node0", 
+                "block_id": 33442,
+                "finality": "optimistic"
+             }'
+```
+
+### Using block hash
+```bash
+http post https://staging-rpc.nearprotocol.com jsonrpc=2.0 id=dontcare \
+    method=query \
+    'params:={
+                "request_type": "view_access_key_list", 
+                "account_id": "node0", 
+                "block_id": 33442,
```

Comment:
It should be a hash here.

P.S. remove `finality` (^^^)

Reply:
✅